### DISCUSSION TITLE
【求助】jm找不到了，天塌了

### DISCUSSION THREAD

**Discussion by dadadeyan** [share121/inter-knot#5718] [公告]
官网进不去，谁能救救我！没有jm晚上我怎么睡觉

**Comment by liang129** [comment]
吓得我跑去看一下，嘿嘿，我还可以打开。

**Comment by 129776** [comment]
嘿嘿，我的还在

**Comment by CeceliaHarex** [comment]
这个是Jm的项目地址[Jmcomic](https://github.com/hect0x7/JMComic-APK/blob/master/README.md)
ps：打不开自备科学上网qwq
